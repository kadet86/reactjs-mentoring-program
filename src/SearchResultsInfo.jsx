// @flow
import * as React from 'react';
import SortBy from './SortBy';
import { Toolbar } from 'primereact/toolbar';

type Props = {
    resultsCount: number,
    sortBy: string,
    onSortByChange: (event: SyntheticEvent<HTMLElement>) => void,
};

export default function SearchResultsInfo({
    resultsCount,
    sortBy,
    onSortByChange,
}: Props) {
    return (
        <Toolbar className="movie-list-toolbar">
            <label className="movie-list-toolbar__label">
                {resultsCount} movies found
            </label>
            <SortBy sortBy={sortBy} onSortByChange={onSortByChange} />
        </Toolbar>
    );
}
