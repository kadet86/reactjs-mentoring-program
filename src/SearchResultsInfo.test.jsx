import React from 'react';
import {shallow} from 'enzyme';
import SearchResultsInfo from './SearchResultsInfo';

describe('SearchResultsInfo component', () => {
    it('has persistent snapshot', () => {
        const component = shallow(<SearchResultsInfo />);
        expect(component).toMatchSnapshot();
    });
});
