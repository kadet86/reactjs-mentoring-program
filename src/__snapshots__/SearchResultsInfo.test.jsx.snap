// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResultsInfo component has persistent snapshot 1`] = `ShallowWrapper {}`;
